app/src/main/java/com/example/tasky/common/presentation/ErrorAlertDialog.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.tasky.common.presentation
+
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.tasky.R
+
+@Composable
+fun CreateErrorAlertDialog(
+    showDialog: MutableState<Boolean>,
```

Comment:
I'd avoid passing Compose states down to Composables like this, but rather only the raw values (just a Boolean here). You won't find any composable from the official lib that will do this, since you give away all the control you have over the state as don't know in how far it can be changed by sub-composables. 

The logic to show/hide a dialog should therefore be hoisted to the screen composable/screen ViewModel to have this logic at a central place which also makes it easier to test.